concur/draw.py

```diff
@@ -194,22 +194,18 @@ def text(string, x, y, color, tf=None):
         yield
 
 
-def image(tex_id, w, h, tf):
+def image(tex_id, w, h, tf, x=0, y=0):
```

Comment:
By convention, `tf` is always the last (optional) argument. Therefore, we can't avoid a breaking change. I am not sure about the ideal argument order. But it is very uncommon to specify origin location after dimensions. _Probably_, it would be better to just do this:

```python
def image(tex_id, x, y, w, h, tf=None):
```